method 'RaisedButton' isn't defined for the class '_MyHomePageState'

I was able to get all of cargo make to work on windows.  I had to add the --no-sound-null-safety as another issue suggested.

Any ideas why I find this RaisedButton isn't defined?

`c:\working\flutter\flutterust>flutter run --no-sound-null-safety
Using hardware rendering with device sdk gphone64 x86 64. If you notice graphics artifacts, consider enabling software rendering with "--enable-software-rendering".
Launching lib\main.dart on sdk gphone64 x86 64 in debug mode...
lib/main.dart:56:13: Error: The method 'RaisedButton' isn't defined for the class '_MyHomePageState'.
 - '_MyHomePageState' is from 'lib/main.dart'.
Try correcting the name to the name of an existing method, or defining a method named 'RaisedButton'.
            RaisedButton(
            ^^^^^^^^^^^^


FAILURE: Build failed with an exception.

* Where:
Script 'C:\tools\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1159

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\tools\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 15s
Running Gradle task 'assembleDebug'...                             16.1s
Exception: Gradle task assembleDebug failed with exit code 1`

--- a/README.md
+++ b/README.md
@@ -92,7 +92,7 @@ cargo make
 Then run flutter app normally
 
 ```
-flutter run
+flutter run --no-sound-null-safety
 ```
 
 ## How it works?

--- a/lib/main.dart
+++ b/lib/main.dart
@@ -53,8 +53,7 @@ class _MyHomePageState extends State<MyHomePage> {
               style: Theme.of(context).textTheme.headline4,
             ),
             const SizedBox(height: 100),
-            RaisedButton(
-              color: Colors.greenAccent,
+            ElevatedButton(
               child: Text(
                 'Scrape rust-lang.org',
                 style: TextStyle(

